src/Network/Algorand/Api.hs

```diff
@@ -7,12 +7,12 @@
 -- See <https://developer.algorand.org/docs/reference/rest-apis/algod/v2/>
```

Comment:
The comment above should be updated btw.

Reply:
done
sorry for squashed commits